src/server/rest/v1/service/TransactionService.ts

```diff
@@ -40,6 +40,28 @@ import moment from 'moment-timezone';
 const MODULE_NAME = 'TransactionService';
 
 export default class TransactionService {
+
+  public static async handleGetTransactions(action: ServerAction, req: Request, res: Response, next: NextFunction): Promise<void> {
+    const filteredRequest = TransactionSecurity.filterGetTransactions(req.query);
+    let transactions: DataResult<Transaction>;
+    if (filteredRequest.Status) {
+      if (filteredRequest.Status === 'completed') {
+        transactions = await TransactionService.getTransactionsCompleted(ServerAction.TRANSACTIONS_COMPLETED, req);
+      } else if (filteredRequest.Status === 'active') {
+        transactions = await TransactionService.getTransactionsActive(ServerAction.TRANSACTIONS_ACTIVE, req);
+      }
+    } else {
+      const completed = await TransactionService.getTransactionsCompleted(ServerAction.TRANSACTIONS_COMPLETED, req);
```

Comment:
This does not work.
You break the pagination.

Reply:
Removed